### ISSUE TITLE
Dark Mode for Desktop App

### ISSUE THREAD

**Issue by Mactastic1-5** [meganz/MEGAsync#829]
The Website has Dark Mode, the Mobile App has Dark Mode. Now it's time for the Desktop App to have Dark Mode on Windows/Mac/Linux.

**Comment by Leonm99** [comment]
i was JUST searching if it had it already, yeah its time for the PC app to get darkmode 100%

**Comment by SJai-mega** [comment]
Hello, Thank you for your feedback. We are in the process on implementing this in a future update. It is coming soon ™
Please keep an eye out for this in the future.

**Comment by eksynn** [comment]
> Hello, Thank you for your feedback. We are in the process on implementing this in a future update. It is coming soon ™ Please keep an eye out for this in the future.

did you guys forget dark mode or something? i hate using it because it always blinds me as soon as it launches - try flashing a really bright flashlight into your eyes with the light on - that's the same feeling >.<
please put some priority on this. an application - especially one as major as mega - with no dark-mode nowadays makes absolutely no sense

**Comment by ccfman2004** [comment]
I have cataracts and cannot read anything not in dark mode anymore.

**Comment by AssassinLV** [comment]
bump.... Why this is not done... This is more than annoying...

**Comment by elkuku** [comment]
Just another bump... dark mode please :wink:

**Comment by AssassinLV** [comment]
Myeah... I think in 4-8 months I will require artificial sight - because Mega would have made me blind.........

**Comment by KiraiEEE** [comment]
no dark mode yet ? can I help?

**Comment by AssassinLV** [comment]
This thing is even not assigned to anyone - so I guess unless we stop paying mega money - they won't care :/

**Comment by KiraiEEE** [comment]
its 2024 and mega has no dark mode imagine

**Comment by Acid-Breath** [comment]
Bro where is the Dark Mode???

**Comment by ozansh** [comment]
I suppose they dont develop it's dark mode anymore :/

**Comment by TheGallant** [comment]
Please where is the Dark mode?!

**Comment by ciros88** [comment]
+1 for dark mode on linux

**Comment by shisuuu** [comment]
I need dark mode, my eyes are bleeding whenever I open the Transfer Manager :(

**Comment by Austinexe** [comment]
is there any update on this by chance?

**Comment by Damians0** [comment]
it's shameful, the mega application for windows still doesn't have dark mode, the developers are making fun of us, I will use another cloud storage service.

**Comment by rautamiekka** [comment]
> it's shameful, the mega application for windows still doesn't have dark mode, the developers are making fun of us, I will use another cloud storage service.

Cut the exaggeration, you're acting like a child for nothing.

**Comment by Psychoticinsane** [comment]
> > it's shameful, the mega application for windows still doesn't have dark mode, the developers are making fun of us, I will use another cloud storage service.
> 
> Cut the exaggeration, you're acting like a child for nothing.

i agree with who you are replying too, more than a year ago in august of 2023 they lied about developing it, and that it was coming soon. im sorry but 15 months is not soon in any way shape or form, and they have been absolutely silent on it. they neve rintended to develop it. maybe you jus need to stay off the internet if people being upset aboutbeing lied too, bothers you

**Comment by Sin-Shadow-Fox** [comment]
Update for anyone that genuinely wants dark mode for the desktop app. Download the mobile app and leave a one star review for that on the play store (keep is clean or no one will see it). Companies don't care what you want but they do care about their ratings. If their mobile app has nothing but bad ratings then they'll start caring real quick.

**Comment by cipricus** [comment]
@Sin-Shadow-Fox Isn't that a bit off given that the mobile app *does* have dark mode?

**Comment by Sin-Shadow-Fox** [comment]
> @Sin-Shadow-Fox Isn't that a bit off given that the mobile app _does_ have dark mode? I find it absurd to down-vote to one star an app that is fine just because a similar one by the same provider doesn't have dark mode. None of their products deserves less than 5 stars (at least from me, I have 50 GB of free cloud from them, and compared to it there is no competition as far as syncing programs go, especially on Linux where Google Drive and OneDrive are not available). For a paying customer which feels entitled to more you should stop paying. Is that your case?

@cipricus It IS extreme, yes. But so is saying your "working" on something as simple as a dark mode for several YEARS and not doing it. It comes across as intentional/malicious so i feel my response is called for, and like i said, affecting someones ratings is the best way to get a companies attention most of the time. Besides, our ratings can be changed after making them should Mega finally implement dark mode for the desktop app. But if you have a better solution, then I'm listening.

**Comment by Sin-Shadow-Fox** [comment]
> > > @Sin-Shadow-Fox Isn't that a bit off given that the mobile app _does_ have dark mode? I find it absurd to down-vote to one star an app that is fine just because a similar one by the same provider doesn't have dark mode. None of their products deserves less than 5 stars (at least from me, I have 50 GB of free cloud from them, and compared to it there is no competition as far as syncing programs go, especially on Linux where Google Drive and OneDrive are not available). For a paying customer which feels entitled to more you should stop paying. Is that your case?
> > 
> > 
> > @cipricus It IS extreme, yes. But so is saying your "working" on something as simple as a dark mode for several YEARS and not doing it. It comes across as intentional/malicious so i feel my response is called for, and like i said, affecting someones ratings is the best way to get a companies attention most of the time. Besides, our ratings can be changed after making them should Mega finally implement dark mode for the desktop app. But if you have a better solution, then I'm listening.
> 
> Not only is that straight-up hypocritical, you could be reported+punished, if not banned, for abusing an unrelated service, in this case Google Play; it's almost directly analogous to vandalizing a Walmart shop which sells a boxed software just cuz the software doesn't look the part, in this case the dark mode. Yours and others' reactions are so extreme it's shameful.

What's shameful is telling people that your "working" on it and it's coming "soon" for several years to the point that people believe that not doing it is intentional/malicious. Also a different version of the same software is hardly "unrelated". I made my point as mega has responded by saying their now looking into it. We'll see if the influx of one star reviews helps motivate them.

**Comment by micnorian14** [comment]
Still no dark mode fr?

**Comment by AssassinLV** [comment]
I hate to do this - but those with paid accounts - I think this is the only way to make the Mega hear us...

![Image](https://github.com/user-attachments/assets/7c168f09-16da-4f59-99a8-f959c93b7995)

**Comment by AssassinLV** [comment]
> 
> They've been saying that since 2023

Hence I did said for hopes of for this century :D

**Comment by AssassinLV** [comment]
Wow - It's "in development" and "will be released soon" since Sept 5 2001......

**Comment by vtmateos** [comment]
Dear Users,

Thank you for your interest in the new dark mode feature. We truly appreciate seeing so many people eager for its release.

We are currently working on enhancing theming capabilities for the desktop app. You may have noticed that, since some recent versions, new code has been added to support these kind of features:

[New theme manager](https://github.com/meganz/MEGAsync/blob/754514cc6f2617413a96fc462ead3810851e6ccc/src/MEGASync/gui/ThemeManager.cpp#L6)
[Design tokens importer](https://github.com/meganz/MEGAsync/tree/master/src/DesignTokensImporter)

There are a large number of UIs and components that need to be adjusted to align with our new design tokens architecture and restyled accordingly. I’d like to get your feedback on which areas you think we should be prioritized for dark mode. Please share your thoughts in this issue.

To wrap up, I’ve attached some screenshots of our first redesigned section (Settings) to give you a glimpse of how it will look like.

Thanks for your support.

![Image](https://github.com/user-attachments/assets/a2e469e9-a38b-4e96-8d9e-492f3ee13afc)
![Image](https://github.com/user-attachments/assets/066d07fe-b410-46d8-840c-c93fc3a2367b)
![Image](https://github.com/user-attachments/assets/2cd8ace0-d4ac-4113-b94b-4479e1b0fe44)

**Comment by Sin-Shadow-Fox** [comment]
> Dear Users,
> 
> Thank you for your interest in the new dark mode feature. We truly appreciate seeing so many people eager for its release.
> 
> We are currently working on enhancing theming capabilities for the desktop app. You may have noticed that, since some recent versions, new code has been added to support these kind of features:
> 
> [New theme manager](https://github.com/meganz/MEGAsync/blob/754514cc6f2617413a96fc462ead3810851e6ccc/src/MEGASync/gui/ThemeManager.cpp#L6) [Design tokens importer](https://github.com/meganz/MEGAsync/tree/master/src/DesignTokensImporter)
> 
> There are a large number of UIs and components that need to be adjusted to align with our new design tokens architecture and restyled accordingly. I’d like to get your feedback on which areas you think we should be prioritized for dark mode. Please share your thoughts in this issue.
> 
> To wrap up, I’ve attached some screenshots of our first redesigned section (Settings) to give you a glimpse of how it will look like.
> 
> Thanks for your support.


If your actually looking for feedback then what you seem to have looks decent enough. If you really wanted to be fancy, you could add a gradient so that it's not all the same shade of grey. You could also do something about the buttons. But if what @AssassinLV said is true, that you've "been working" on this since 2001 then non of us believe you or these screenshots. We won't believe you until the next update actually implements these changes.

**Comment by AssassinLV** [comment]
> 
> If your actually looking for feedback then what you seem to have looks decent enough. If you really wanted to be fancy, you could add a gradient so that it's not all the same shade of grey. You could also do something about the buttons. But if what [@AssassinLV](https://github.com/AssassinLV) said is true, that you've "been working" on this since 2001 then non of us believe you or these screenshots. We won't believe you until the next update actually implements these changes.

Sorry - fixed typo - 2021 :D
https://github.com/meganz/MEGAsync/issues/618

**Comment by AssassinLV** [comment]
3 months later........... Same as in 2021 ......

**Comment by curtainteddy** [comment]
![It's been 84 years...](https://c.tenor.com/R6D5FUXHb2IAAAAd/tenor.gif)

**Comment by OneTimePads** [comment]
We gonna get it in next few years ... I can feel it in my old bones ...

**Comment by AssassinLV** [comment]
> We gonna get it in next few years ... I can feel it in my old bones ...

Yes, we will get it, but around the time of GTA VIII, if we consider how fast Mega is being updated......

**Comment by portable-hole** [comment]
There was a new version of MegaSync today. I got my hopes up. No Dark Mode.

**Comment by ciros88** [comment]
> ![Image](https://github.com/user-attachments/assets/786c638b-a7de-4fb1-81f4-d669d98cdceb)

try this :)

**Comment by portable-hole** [comment]
> > ![Image](https://github.com/user-attachments/assets/786c638b-a7de-4fb1-81f4-d669d98cdceb)
> 
> try this :)

Done.

**Comment by BinhPQ2** [comment]
Seriously, is adding a new background and a new text colour that much of a work :v

**Comment by noembryo** [comment]
> try this :)

Another issue reported..

**Comment by ForzonLazure** [comment]
Still no updates?

**Comment by portable-hole** [comment]
Project has been abandoned.

**Comment by vtmateos** [comment]
Dear users,

Thank you for your patience, and please accept our apologies for the delay in bringing you dark mode support.
We have been working diligently to provide you with a new and improved dark mode experience, and we are now in the final stages of testing. The release is very close.

We kindly ask for just a little more patience as we make sure everything is ready for you.

Kind regards.

**Comment by vtmateos** [comment]
Update: Dark mode support is officially available since 6.0.0 version. You can get latest installer from our website and autoupdate will reach you in the next days. Linux still a bit of time to polish it.
Regards.

**Comment by portable-hole** [comment]
It's glorious!

> Update: Dark mode support is officially available since 6.0.0 version. You can get latest installer from our website and autoupdate will reach you in the next days. Linux still a bit of time to polish it. Regards.

**Comment by t0ma5** [comment]
let's goooooooooooooooooooooooooooo! I waited for YEARS for this to be implemented! Thank u :)
